src/WebJobs.Script/OutOfProc/Rpc/LanguageWorkerProcessFactory.cs

```diff
@@ -38,11 +38,11 @@ internal class LanguageWorkerProcessFactory : ILanguageWorkerProcessFactory
             _processRegistry = processRegistry;
         }
 
-        public ILanguageWorkerProcess CreateLanguageWorkerProcess(string workerId, string runtime, string scriptRootPath)
+        public ILanguageWorkerProcess Create(string workerId, string runtime, string scriptRootPath)
         {
-            WorkerConfig workerConfig = _workerConfigs.Where(c => c.Language.Equals(runtime, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
+            WorkerConfig workerConfig = _workerConfigs.Where(c => c.Description.Language.Equals(runtime, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
             ILogger workerProcessLogger = _loggerFactory.CreateLogger($"Worker.LanguageWorkerProcess.{runtime}.{workerId}");
-            return new LanguageWorkerProcess(runtime, workerId, scriptRootPath, _rpcServer.Uri, workerConfig.Arguments, _eventManager, _workerProcessFactory, _processRegistry, workerProcessLogger, _consoleLogSource);
+            return new RpcWorkerProcess(runtime, workerId, scriptRootPath, _rpcServer.Uri, workerConfig.Arguments, _eventManager, _workerProcessFactory, _processRegistry, workerProcessLogger, _consoleLogSource);
```

Comment:
Rename the type from `LanguageWorkerProcessFactory` to `RpcWorkerProcessFactory`